wordpress-mobile/gutenberg-mobile
# Fix cell alignment on image settings for Android

This PR updates the GB reference where the alignment issue is fixed.

To test:
 - Start the demo app in Android
 - Open the image selection options
 - Check if bottom sheet show options left aligned
 - Check other uses of bottom sheets in settings and alignment options

PR submission checklist:
